src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -266,7 +266,9 @@ suggestAddExtension Diagnostic{_range=_range,..}
 
 -- | All the GHC extensions
 ghcExtensions :: Map.HashMap T.Text Extension
-ghcExtensions = Map.fromList . map ( ( T.pack . flagSpecName ) &&& flagSpecFlag ) $ xFlags
+ghcExtensions = Map.fromList . filter notStrictFlag . map ( ( T.pack . flagSpecName ) &&& flagSpecFlag ) $ xFlags
+  where
+    notStrictFlag (name, _) = name /= "Strict"
```

Comment:
I'd add a link to the issue you initially raised here. It's one odd and unexpected exception, so having the backstory is always useful.

Reply:
@ndmitchell done!
there's no issue though, only this PR.

Reply:
Thanks - this PR is fine.